src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: bold;
+  font-style: normal;
+}
+
+.container {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big {
+  height: 66px;
+  font-weight: 300;
```

Comment:
The same with .small input, please move to part of code where you initializate font
![image](https://user-images.githubusercontent.com/54936044/231728965-c3c1a37e-aeca-4fa1-a5c2-57b363512934.png)
![image](https://user-images.githubusercontent.com/54936044/231729612-607ce0e3-6918-4d7c-946f-94bdcf64849d.png)